### PR TITLE
Fix build with cozy permissions

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: test/helpers/password.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 // The password used for integration tests:
-let password = 'CozyTest$1'
+let password = process.env.COZY_PASSPHRASE || 'CozyTest_1'
```
Comment (sebn) [inline test/helpers/password.js]: The goal of this "executable" module was for the default test password/passphrase to be defined in a single place, but now it is defined thrice.

File: src/app.js [hunk 2]
```diff
@@ -152,14 +152,14 @@ class App {
       return
     }
     if (deviceName == null) { deviceName = os.hostname() || 'desktop' }
-    this.askPassword(function (_, password) {
+    this.askPassword(function (_, passphrase) {
```
Comment (sebn) [inline src/app.js]: `s/askPassword/askPassphrase`? (can be done later)

File: test/helpers/password.js (hunk 1 above)
Comment (sebn) [inline test/helpers/password.js]: Or do you think of some scenario where still being able to overwrite it is more important?

**Comment by sebn** [comment]
Merging as soon as CI is green (unless you want me to wait).

File: test/helpers/password.js (hunk 1 above)
Comment (nono) [inline test/helpers/password.js]: Don't think too much about this. I think this module will disappear. We will use a token generated by the stack, not the passphrase, in tests.

**Comment by sebn** [comment]
Strange issue: when using `yarn link cozy-client-js`, [this line](https://github.com/nono/cozy-desktop/blob/f8655c96d371f20077d3585c0392f07792a5666a/test/helpers/cozy.js#L22) fails because `CozyClient` is undefined. As soon as I unlink and install version 0.1.0 it works.

Since we don't use `yarn link` on Travis it should work.

**Pull request merged**